wiki/Game_mode/osu!taiko/en.md

```diff
@@ -12,7 +12,7 @@ osu!taiko is a game mode in osu! based on the Japanese rhythm game series, [Taik
 
 ### Song Selection
 
-To access the osu!taiko game mode, press `Ctrl` + `2` at the same time.
+To access the osu!taiko game mode, press `Ctrl` &  `2` at the same time.
```

Comment:
Is this really necessary?

Reply:
in my opinion it flows better

Reply:
most of the cases, mostly its mentioned with `+` than `&`

Reply:
using `&` here is indeed wrong.

Reply:
will change

Reply:
done